src/Configuration/Accounts.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ClassicUO.Configuration
+{
+    internal class Account 
+    {
+        public Account()
+        {
+
```

Comment:
Sorry for being picky regarding formatting. Can you please remove the empty line here?
(SonarLint/Cube/StyleCop will cry about it, if we decide to add it :D)

Reply:
done